.github/workflows/prod.yml

```diff
@@ -14,27 +14,14 @@ jobs:
     steps:
     - name: Checkout sources
       uses: actions/checkout@v2
-    - name: Test wrapper
+    - name: Test use Gradle wrapper
       uses: ./
       with:
-        wrapper-directory: __tests__/data/basic
```

Comment:
:x: This removes test coverage for the `wrapper-directory` option. We should keep some coverage.

I understand those changes make this option unnecessary but it would be good to publish a `v1` update with this fix. I'd be more confortable doing this with some test coverage.